TAO/tests/Oneway_Send_Timeouts/ORB_Task.h

```diff
@@ -7,22 +7,22 @@
 
 class ORB_Task : public ACE_Task_Base
 {
- private:
+private:
   CORBA::ORB_var orb_;
 
- public:
+public:
   ORB_Task (CORBA::ORB_ptr orb)
     : orb_ (CORBA::ORB::_duplicate (orb))
-    {
-      if (CORBA::is_nil (orb_.in()) == 1) {
-  ACE_ERROR ((LM_ERROR, "ORB_Task> Ctr> Orb is NULL\n"));
-      }
-    };
+  {
+    if (CORBA::is_nil (orb_.in ()) == 1) {
```

Comment:
Normal ACE coding style would be `{` on the next line, but we have so many older tests contributed by a lot of people that coding styles are different

Reply:
No Problem, can be changed

Reply:
@jwillemsen Q: what should be the value for `ColumnLimit: 79` ?

Reply:
That is very small, would go for at least 120, maybe even 160

Reply:
@jwillemsen Q: and what is about this? `AlwaysBreakAfterReturnType: None`

Reply:
Does https://htmlpreview.github.io/?https://github.com/DOCGroup/ACE_TAO/blob/master/ACE/docs/ACE-guidelines.html help?

Reply:
**Yes**! But **No** in respect to `AlwaysBreakAfterReturnType: None` the examples are inconsistent!

Reply:
I think we should use `RTBS_All`